### ISSUE TITLE
No FAQ section

### ISSUE THREAD

**Issue by ladyami** [Open-Science-Community-Saudi-Arabia/OSCSA_Website#77]
some user would want to know a little info  the help FAQ section will help them out.

**Comment by BatoolMM** [comment]
Thank you for your suggestions @ladyami. Do you have FAQs in mind?

**Comment by AminMusah** [comment]
@ladyami @BatoolMM i do have some FAQs in mind, can i work on it?

**Comment by ladyami** [comment]
yes @AminMusah I have code the FAQ Just  the questions and answer to fill it

**Comment by AminMusah** [comment]
sure i have the question and answer to fill it, i can help you with that.

**Comment by ladyami** [comment]
@AminMusah  please open a issue and submit a pr for it
